File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float input;
    int change;
    int tc = 0; //total coins
    do
    {
        input = get_float("Change owed: ");
    }
    while (input < 0);
    change = round(input * 100);
    
    while (change > 0)
    {
        if (change >= 25)
        {
            change -= 25;
            tc++;
        }
        else if (change >= 10)
        {
            change -= 10;
            tc++;
        }
        else if (change >= 5)
        {
            change -= 5;
            tc++;
        }
        else
        {
            change--;
            tc++;
        }
    }
    printf("Total coins: %i\n", tc);
}

File: pset6/p3_cash/cash.py
from cs50 import get_float


def main():

    while True:
        n = get_float("Change owed: ")
        if n > 0:
            break

    n = round(n * 100)
    tc = 0

    while n > 0:
        if n >= 25:
            n -= 25
            tc += 1

        elif n >= 10:
            n -= 10
            tc += 1

        elif n >= 5:
            n -= 5
            tc += 1

        else:
            n -= 1
            tc += 1

    print("Total Coins", tc)


main()

# check50 cs50/problems/2021/x/sentimental/cash
# submit50 cs50/problems/2021/x/sentimental/cash